components/header_footer_template/header_footer_template.test.tsx

```diff
@@ -91,22 +90,24 @@ describe('components/HeaderFooterTemplate', () => {
 
     test('should set classes on body and #root on mount', () => {
         expect(document.body.classList.contains('sticky')).toBe(false);
-        expect(document.getElementById('root').classList.contains('container-fluid')).toBe(true);
+        const rootElement: HTMLElement | null = document.getElementById('root');
+        expect(rootElement && rootElement.classList.contains('container-fluid')).toBe(true);
         shallow(<NotLoggedIn config={{AboutLink: 'http://testaboutlink'}}/>);
         expect(document.body.classList.contains('sticky')).toBe(true);
-        expect(document.getElementById('root').classList.contains('container-fluid')).toBe(true);
+        expect(rootElement && rootElement.classList.contains('container-fluid')).toBe(true);
     });
 
     test('should unset classes on body and #root on unmount', () => {
         expect(document.body.classList.contains('sticky')).toBe(false);
-        expect(document.getElementById('root').classList.contains('container-fluid')).toBe(true);
+        const rootElement: HTMLElement | null = document.getElementById('root');
+        expect(rootElement && rootElement.classList.contains('container-fluid')).toBe(true);
         const wrapper = shallow(
             <NotLoggedIn config={{AboutLink: 'http://testaboutlink'}}/>,
         );
         expect(document.body.classList.contains('sticky')).toBe(true);
-        expect(document.getElementById('root').classList.contains('container-fluid')).toBe(true);
+        expect(rootElement && rootElement.classList.contains('container-fluid')).toBe(true);
         wrapper.unmount();
         expect(document.body.classList.contains('sticky')).toBe(false);
-        expect(document.getElementById('root').classList.contains('container-fluid')).toBe(false);
+        expect(rootElement && rootElement.classList.contains('container-fluid')).toBe(false);
```

Comment:
you can shorten this to: 
```suggestion
        expect(rootElement?.classList?.contains('container-fluid')).toBe(false);
```
(i think :) )